Dockerfile

```diff
@@ -1,10 +1,27 @@
+FROM golang:latest AS build
+
+RUN mkdir -p /go/src/github.com/nsqio/nsq
+COPY . /go/src/github.com/nsqio/nsq
+
+WORKDIR /go/src/github.com/nsqio/nsq
+
+RUN go get github.com/mattn/goveralls \
```

Comment:
goveralls isn't needed here since ./coverage.sh is not run here

Reply:
You're right, yeah...